bin/run-versioned-tests.sh

```diff
@@ -3,13 +3,9 @@
 set -x
 VERSIONED_MODE="${VERSIONED_MODE:---minor}"
 SAMPLES="${SAMPLES:-15}"
+C8_REPORTER="${C8_REPORTER:-lcov}"
```

Comment:
I don't see you overriding this in CI with `lcovonly`. I assume that was your intention correct?

Reply:
Ah, yes, I need to update the workflow to include the env var. I also noticed that we're not currently providing the minor/major env var, so we're always running with the `--minor` flag even on PRs, do you think I should change it so we run on PRs with `--major`?

Reply:
That was done purely to cut down on time in the main agent.  I'll leave that up to you to decide

Reply:
I ended up adding the VERSIONED_MODE flag in after all, without it the PR build was [20 minutes](https://github.com/newrelic/newrelic-node-apollo-server-plugin/actions/runs/5137181985/jobs/9244937127), which seemed a bit long imo.